### ISSUE TITLE
Gitlab CI/CD support 

### ISSUE THREAD

**Issue by vipafattal** [nhost/nhost#2299]
Is there any plan to support Gitlab CI/CD ?

**Comment by dbarrosop** [comment]
What do you mean with "Gitlab CI/CD"? Do you want to connect a project to gitlab or do you want to be able to deploy yourself with your own workflow?

**Comment by vipafattal** [comment]
Yes, I want to use Gitlab as a repository and connect to Nhost Cloud instead of Github

**Comment by dbarrosop** [comment]
Ok, it is in our backlog but we don't have a timeframe yet.

**Comment by fmanzali** [comment]
As a workaround, you can create a (free) repo on GitHub, then configure mirroring from GitLab to GitHub, then use the git integration with Nhost.
